crates/biome_js_formatter/src/js/bindings/parameters.rs

```diff
@@ -109,6 +150,29 @@ impl Format<JsFormatContext> for FormatAnyJsParameters {
                     write!(f, [format_removed(&r_paren_token)])?;
                 }
 
+                Ok(())
+            }
+            ParameterLayout::Compact => {
+                if !parentheses_not_needed {
+                    write!(f, [l_paren_token.format()])?;
+                } else {
+                    write!(f, [format_removed(&l_paren_token)])?;
+                }
```

Comment:
Seems like we could probably move these paren token writes outside of the match and reduce deduplication, but not gonna do that for now.